### PR TITLE
tests: fix e2e tests for cm alerts

### PR DESCRIPTION
**Pull request by gssbzn**
## Checklist

- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have updated [e2e/E2E-TESTS.md](e2e/E2E-TESTS.md) (if a new command or e2e test has been added)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

File: e2e/cloud_manager/alerts_test.go [hunk 1]
```diff
@@ -37,15 +38,13 @@ func TestAlerts(t *testing.T) {
 		t.Fatalf("unexpected error: %v", err)
 	}
 
-	const alertsEntity = "alerts"
-	alertID := "5ec2ac941271767f21cbaefe"
+	var alertID string
 
-	t.Run("Describe", func(t *testing.T) {
+	t.Run("List with no status", func(t *testing.T) {
```
Comment (gssbzn) [inline e2e/cloud_manager/alerts_test.go:43]: move list first to avoid hardcoding the describe id

File: e2e/cloud_manager/alerts_test.go [hunk 2]
```diff
@@ -145,10 +144,6 @@ func TestAlerts(t *testing.T) {
 		if err := json.Unmarshal(resp, &alerts); err != nil {
 			t.Fatalf("unexpected error: %v", err)
 		}
-
-		if len(alerts.Results) > 0 {
```
Comment (gssbzn) [inline e2e/cloud_manager/alerts_test.go:149]: remove this count, we got a new alert

File: e2e/cloud_manager/alerts_test.go [hunk 3]
```diff
@@ -204,7 +199,7 @@ func TestAlerts(t *testing.T) {
 		}
 	})
 
-	t.Run("UnaAcknowledge", func(t *testing.T) {
```
Comment (colm-quinn) [inline e2e/cloud_manager/alerts_test.go:207]: 👁️

**Review by colm-quinn — APPROVED** [review]
LGTM if LGTEvg

**Pull request merged**